src/megachatapi_impl.cpp

```diff
@@ -2022,15 +2022,14 @@ void MegaChatApiImpl::sendPendingRequests()
             bool hasReacted = msg.hasReacted(reaction, mClient->myHandle());
             if (request->getFlag())
             {
-//                Uncomment this block upon release of reaction limitations
-//                // check if max number of reactions has been reached
-//                int res = msg.allowReact(mClient->myHandle(), reaction);
-//                if (res != 0)
-//                {
-//                    request->setNumber(res);
-//                    errorCode = MegaChatError::ERROR_NOENT;
-//                    break;
-//                }
+                // check if max number of reactions has been reached
+                int res = msg.allowReact(mClient->myHandle(), reaction);
+                if (res != 0)
+                {
+                    request->setNumber(res);
+                    errorCode = MegaChatError::ERROR_NOENT;
```

Comment:
Could we return a new error code that doesn't clash with this one, already used for non existing `msgid`??
Perhaps a `ERROR_TOOMANY` (we can define it, in case it doesn't exist yet).

Reply:
I returned ERROR_NOENT as ticket specified, but there's no problem to change it.

Reply:
Fixed in db87993eda1bbe0cc1cc71fcc19de463e7e9e252